src/main/java/at/hannibal2/skyhanni/features/garden/composter/ComposterOverlay.kt

```diff
@@ -414,8 +418,35 @@ class ComposterOverlay {
             list.add(Renderable.link("$name§r §8x${itemsNeeded.addSeparators()} §7(§6$format§7)") {
                 onClick(internalName)
                 if (LorenzUtils.isControlKeyDown()) {
-                    inInventory = false
-                    BazaarApi.searchForBazaarItem(itemName, itemsNeeded.toInt())
+                    if(lastAttemptTime.passedSince()>0.5.seconds){
```

Comment:
Isn't `Renderable.link` already an edge trigger? Any reason to have another check to prevent spamming.